### PR TITLE
Add pytwin to pyansys metapackage.

### PR DESCRIPTION
**Pull request by MaxJPRey**
Adding PyTwin to PyAnsys metapackage

### CODE REVIEW

File: doc/source/index.rst [hunk 1]
```diff
@@ -111,3 +111,12 @@ it is now a collection of many Python packages for using Ansys products through
       :class-title: pyansys-card-title
 
       Shared software components to enable package interoperability and minimize maintenance
+
```
Comment (RobPasMue) [inline doc/source/index.rst:114]:
```suggestion
```

**Review by RobPasMue — APPROVED** [review]
LGTM! One minor comment - I'll merge it myself

**Comment by RobPasMue** [comment]
@MaxJPRey looks like Python 3.7 is not supported by PyTwin... they might want to solve that. Python 3.7 is maintained until mid 2023. So we should still support it.

File: README.rst [hunk 2]
```diff
@@ -48,6 +48,7 @@ At this moment, this package ensures the compatibility between the following PyA
 - `PySeascape <https://seascape.docs.pyansys.com/>`_: Pythonic interface to communicate with RedHawkSC and TotemSC
 - `Granta MI BoM Analytics <https://grantami.docs.pyansys.com/>`_: Pythonic interface to Granta MI BoM Analytics services
 - `Shared Components <https://shared.docs.pyansys.com/>`_: Shared software components to enable package interoperability and minimize maintenance
+- `PyTwin <https://shared.docs.pyansys.com/>`_: PyTwin is a Python package that eases Ansys Digital Twins consumption workflows
```
Comment (RobPasMue) [inline README.rst:51]:
```suggestion
- `PySeascape <https://seascape.docs.pyansys.com/>`_: Pythonic interface to communicate with RedHawkSC and TotemSC
- `PyTwin <https://shared.docs.pyansys.com/>`_: PyTwin is a Python package that eases Ansys Digital Twins consumption workflows
- `Granta MI BoM Analytics <https://grantami.docs.pyansys.com/>`_: Pythonic interface to Granta MI BoM Analytics services
- `Shared Components <https://shared.docs.pyansys.com/>`_: Shared software components to enable package interoperability and minimize maintenance
```

File: README.rst [hunk 3]
```diff
@@ -65,6 +66,7 @@ By default, the PyAnsys package installs these core modules:
 - `PySeascape`_
 - `Granta MI BoM Analytics`_
 - `Shared Components`_
+- `PyTwin`_
```
Comment (RobPasMue) [inline README.rst:69]:
```suggestion
- `PySeascape`_
- `PyTwin`_
- `Granta MI BoM Analytics`_
- `Shared Components`_
```

**Comment by RobPasMue** [comment]
I am going to reorganize the tabs a little bit so that they go right after PySeascape

**Comment by RobPasMue** [comment]
OKay so, right now our biggest issue is the fact that it does not support Python 3.7. I think that I can come around it for now... give me some minutes. But they should definitely try to fix it.

**Comment by MaxJPRey** [comment]
@lboucin I created the image `pytwin.png` from some an image located in PyTwin documentation.
Is it OK with you to represent PyTwin with this one in our cards documentation: https://docs.pyansys.com/ ?

**Comment by lboucin** [comment]
Hello @MaxJPRey, I'll double check with my product team and confirm we can use the `pytwin.png` you've created. Thanks!

**Comment by RobPasMue** [comment]
Hi @lboucin! Do you have any news regarding the image?

**Comment by lboucin** [comment]
Hi @RobPasMue, we have an updated pytwin.png image that I'll share with you this morning.

**Comment by RobPasMue** [comment]
Awesome, whenever you have it, just send it to me =)

**Comment by RobPasMue** [comment]
Now that the image is update, we are good to go! Merging as soon as the workflow passes.

**Pull request merged**